indico/modules/events/editing/client/js/components/FileManager/FileManager.module.scss

```diff
@@ -24,15 +24,25 @@
     }
 }
 
+.file-manager-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
 .file-manager {
     display: flex;
+    flex-wrap: wrap;
 
     .file-type {
+        &:not(:nth-child(3)) {
+            margin-right: 1em;
+        }
+
+        margin-bottom: 1em;
         border: 1px solid $gray;
         border-radius: 3px;
         padding: 1em;
-        margin: 1em;
-        width: 33%;
+        flex: 0 0 32%;
```

Comment:
Why do we use 32% instead of 33% here?

Reply:
Because of the margin. With 33% there will be usually two items per row.